src/ucp/wireup/select.c

```diff
@@ -1189,6 +1203,7 @@ ucp_wireup_add_amo_lanes(const ucp_wireup_select_params_t *select_params,
     ucp_wireup_init_select_flags(&criteria.local_iface_flags,
                                  UCT_IFACE_FLAG_PENDING, 0);
     ucp_wireup_fill_peer_err_criteria(&criteria, ep_init_flags);
+    ucp_wireup_fill_exported_memh_criteria(context, &criteria);
```

Comment:
no need for amo

Reply:
let's keep it for future and consistency for now.
also, we shouldn't select AM emulation for AMO in case of EXPORTED_MEMH feature was requested

Reply:
we shouldn't select AM emulation for AMO in case of EXPORTED_MEMH feature was requested

Reply:
can you elaborate why?